models/linked_list.py

```diff
@@ -0,0 +1,140 @@
+from traceback import print_list
+from models.sort_order import SortOrder
+from models.node import Node
+
+
+class ResistorLinkedList:
```

Comment:
Could be just a general LinkedList implementation, no need to name it ResistorLinkedList, but OK.

Reply:
Renamed class